src/sentry/eventstream/kafka.py

```diff
@@ -14,12 +14,6 @@
 logger = logging.getLogger(__name__)
 
 
-# Beware! Changing this, or the message format/fields themselves requires
-# consideration of all downstream consumers.
-# Version 0 format: (0, '(insert|delete)', {..event json...})
-EVENT_PROTOCOL_VERSION = 0
```

Comment:
Psssh, don't like my warning? :)

Reply:
I think initially I thought I would need to support old versions for some reason so having a single constant didn't make sense. I'll bring it back